govcd/catalog_test.go

```diff
@@ -84,3 +96,179 @@ func (vcd *TestVCD) Test_DeleteCatalog(check *C) {
 	check.Assert(catalog, Equals, Catalog{})
 
 }
+
+func (vcd *TestVCD) Test_UploadOvf(check *C) {
+	checkUploadOvf(vcd, check, getCurrentPath()+OvaPath, TestCreateCatalog, "item1")
+}
+
+func (vcd *TestVCD) Test_UploadOvf_chuncked(check *C) {
+	checkUploadOvf(vcd, check, getCurrentPath()+OvaChunkedPath, TestCreateCatalog+"2", "item2")
+}
+
+func (vcd *TestVCD) Test_UploadOvf_progress_works(check *C) {
+	catalogName := TestCreateCatalog + "3"
+	itemName := "item3"
+	setupCatalog(vcd, check, catalogName)
+
+	catalog, org := findCatalog(vcd, check, catalogName)
+	AddToCleanupList(itemName, "catalogItem", vcd.org.Org.Name, "Test_UploadOvf")
+
+	//execute
```

Comment:
Unnecessary comment.  (Same applies to verify below.)

Reply:
Removed.